src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/FundingComplete.cshtml

```diff
@@ -0,0 +1,22 @@
+﻿@model SFA.DAS.EmployerAccounts.Web.ViewModels.AccountDashboardViewModel
+
+@if (Model.ReservedFundingToShow != null)
+{
+    <h3 class="das-panel__heading">Apprenticeship funding secured</h3>
+    <dl class="das-definition-list das-definition-list--with-separator">
+        <dt class="das-definition-list__title">Legal entity:</dt>
+        <dd class="das-definition-list__definition">@Model.ReservedFundingToShowLegalEntityName</dd>
+        <dt class="das-definition-list__title">Training course:</dt>
+        <dd class="das-definition-list__definition">@Model.ReservedFundingToShow.CourseName</dd>
+        <dt class="das-definition-list__title">Start and end date: </dt>
+        <dd class="das-definition-list__definition">@Model.ReservedFundingToShow.StartDate.ToString("MMMM yyyy") to @Model.ReservedFundingToShow.EndDate.ToString("MMMM yyyy")</dd>
+    </dl>
+}
+
+@if (Model.RecentlyAddedReservationId != null &&
```

Comment:
Can this logic be put in the view model so we just use a bool here (e.g. Model.PendingFunding)